mbofui/.gitignore

```diff
@@ -1 +1,2 @@
 bower_components
+.DS_Store
```

Comment:
I think this whole file isn't needed, since one already exists in the parent directory.  Add `bower_components` (with necessary path components) to the other file instead.